### ISSUE TITLE
rename	GO:0031074	nucleocytoplasmic shuttling complex

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#13354]
GO:0031074	nucleocytoplasmic shuttling complex
Any complex that acts to move proteins or RNAs into or out of the nucleus through nuclear pores.

"nucleocytoplasmic shuttling protein" is normally used to describe proteins which are 'shuttled', not the transport complexes. I have community curators using for "shuttled proteins".

 - [x] Could this grouping term be renamed  "nucleocytoplasmic transport complex"? 
- [x] Moves in def should change to transports"
- [ ]  flagged term not for direct annotation"

Some of the  descendent complex terms have "shuttled" in the definition. This should probably also be "transports".  Usually the transport complex is itself "shuttled" but the transport complex is not "shuttling". "Shuttled" implies the complex moves the cargo in both directions, but here is used to describe unidirectional cargo.

**Comment by bmeldal** [comment]
I agree, it should be "transport".

**Comment by pgaudet** [comment]
Hi @ValWood , 

I made the suggested changes, but I have doubt about the 'do not manually annotate' flag. This terms has 4 children, 3 of which seem suspicious to me: 
1. GO:0042564 NLS-dependent protein nuclear import complex: seems to be the same as the parent, and to also include the target of the transport. That seems wrong according to our annotation guidelines. I would be tempted to obsolete this term. 
2. GO:1990602 importin alpha-subunit nuclear import complex
and
3. GO:0042563 importin alpha-subunit nuclear export complex 
seem to be specifically involved in transporting importin alpha back to the cytoplasm or to the nucleus. Again the distinction seems to be the target. 

However once you remove these terms you are left with ' GO:0042565 RNA nuclear export complex', which would mean that we either need a 'protein nuclear import complex' or the parent should be allowed to be used for annotation (or the entire branch could be obsoleted, this all seems very function-like. 

What do you think ? 

Pascale

**Comment by ValWood** [comment]
I agree.
None of the terms should mention the cargo, that would be problematic.
The import receptor complexes appear to be importin alpha plus importin beta,  the beta subunit exchanges with the Ran GTPase. 
Export receptors are exportin with or without Ran.

Because of the combinatorial explosion possible here  with different importins, exportins and Rans perhaps a  single complex term is sufficient in GO,  using protein binding to represent the various combinations. I  would represent  most hetero-dimers using protein binding, unless there is some compelling reason to instantiate  them due to the ancestral nature of the complex (i.e subunits conserved 1:1 across species).

......the import/export  activity can be captured adequately by the functions 
GO:0005049 nuclear export signal receptor activity 
GO:0061608	nuclear import signal receptor activity
and related process terms.

**Comment by bmeldal** [comment]
I concur, just released the Importins in the complex portal!
(Search with GO:0031074 and you'll get them all.)

**Comment by pgaudet** [comment]
Hello @bmeldal @ValWood 

Should I create an issue to obsolete the following terms? 
- GO:0042564 NLS-dependent protein nuclear import complex
- GO:1990602 importin alpha-subunit nuclear import complex
and
- GO:0042563 importin alpha-subunit nuclear export complex

Thanks, Pascale

**Comment by ValWood** [comment]
makes sense to me...

**Comment by bmeldal** [comment]
Fine with me. Thanks.

**Comment by ValWood** [comment]
Hi,

Based on this discussion, should the do not annotate flag be removed from this term?
Or, do we at least need generic complexes for export and import?
Val

**Comment by pgaudet** [comment]
Hi Val, 

The current structure is: 
'nucleocytoplasmic transport complex'
-- 'NLS-dependent protein nuclear import complex'
-- 'RNA nuclear export complex'

The parent has a 'do not annotate' tag. It seems to me one should be able to tell if we're looking at nuclear import of protein or nuclear export of RNA. What do you think ? 

Pascale

**Comment by ValWood** [comment]
there are also complexes to export proteins which shuttle from cytoplasm to nucleus back to cytoplasm.

These results indicated that, as demonstrated for human Pat1b protein (Marnef et al. 2012), Pdc2 was likely a nuclear-cytoplasmic shuttling protein playing a role in translocating Lsm1 from nucleus to cytoplasm. In supporting this notion and in agreement with a previously described systematic localisome study (Matsuyama et al. 2006), we found that Pdc2 interacted with the Crm1, a nuclear exportin involved in protein export (Fig. 5E), and both Pdc2 and Lsm1 accumulated in the nucleus when nuclear ex- port was inhibited by leptomycin B (LMB) (Fig. 5F)

so maybe we just need an additional term?

**Comment by pgaudet** [comment]
The complex would be Pdc2 + Crm1 ?

**Comment by deustp01** [comment]
My recollection from 15 years ago is that three processes can be distinguished: moving proteins from the cytosol into the nucleus via the nuclear pore, moving proteins from the nucleus into the cytosol via the nuclear pore, and moving various kinds of RNAs from the nucleus into the cytosol via the nuclear pore. The differentia are the identities of the proteins mediating the three kinds of movement (some overlap but also some distinct participants) and the identities of the cargoes that are moved (not annotated, I know, but implicitly present and central to defining the larger biological process the specific movement is part of). That sounds like a justification for three GO terms to be used for annotation. Each could be subdivided because different kinds of proteins and RNAs are moved by variants of the relevant generic movement complex. I suspect that this is too close to making terms for specific proteins in the case of protein movement but might be acceptable to distinguish export of mRNA from other RNA classes.

**Comment by ValWood** [comment]
Yes, I agree.

I also deleted the annotation. A community curator made the annotation. I questioned it because I didn't intend to make complex terms for  protein transporter and target (as you say). 
But then I ended up leaving the annotation at the parent term (he actually annotated another protein but I'm not sure that it is a single complex from the data).

So today I deleted it.  We can close this. There clearly are complexes that are transported out of the nucleus, but this isn't a good example.
